msp/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mygovbc-msp",
-  "version": "3.4.2",
+  "version": "3.4.3",
```

Comment:
You'll need to include the update to "package-lock.json" to use the updated version (run "npm install"). I think OpenShift might throw an error if this gets deployed as is.